BundID/Managers/StorageManager.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+class StorageManager: StorageManagerType {
```

Comment:
can be a struct!?

Reply:
We get a problem when updating the setupCompleted property on the StorageManager in the Reducer, because the environment is a read-only struct. I guess my decision to design the API of the StorageManager to look like state was a wrong one (property with get/set), because the environment should be stateless afaik.

I will try to split it into an update method and getter. If you have a better idea, ping me.